samza-core/src/main/scala/org/apache/samza/storage/TaskStorageCommitManager.java

```diff
@@ -101,33 +96,26 @@ public void start() {
       }
     });
 
-    return getStoreSCMs(taskName, backendFactoryStoreStateMarkers);
+    return backendFactoryStoreStateMarkers;
   }
 
-  // TODO HIGH dchen add javadocs explaining what the params are.
-  public void cleanUp(CheckpointId checkpointId, Map<String, List<StateCheckpointMarker>> stateCheckpointMarkers) {
-    // { state backend factory -> { store name -> state checkpoint marker)
-    Map<String, Map<String, StateCheckpointMarker>> stateBackendToStoreSCMs = new HashMap<>();
-
-    // The number of backend factories is equal to the length of the stateCheckpointMarker per store list
-    stateBackendToBackupManager.keySet().forEach((stateBackendFactoryName) -> {
-      stateBackendToStoreSCMs.put(stateBackendFactoryName, new HashMap<>());
-    });
-
-    stateCheckpointMarkers.forEach((storeName, scmList) -> {
-      scmList.forEach(scm -> {
-        if (stateBackendToStoreSCMs.containsKey(scm.getStateBackendFactoryName())) {
-          stateBackendToStoreSCMs.get(scm.getStateBackendFactoryName()).put(storeName, scm);
-        } else {
-          LOG.warn("Ignored cleanup for scm: {} due to unknown factory: {} ", scm, scm.getStateBackendFactoryName());
+  /**
+   * Cleanup  each of the task backup managers
+   * @param checkpointId CheckpointId of the most recent successful commit
+   * @param stateCheckpointMarkers map of map(stateBackendFactoryName to map(storeName to StateCheckpointMarkers) from
```

Comment:
Fix param description. Will be more readable to use the format you used in JsonCheckpoint.